src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1649,6 +1649,17 @@ hibernate.connection.url=jdbc:hsqldb:mem:rm;hsqldb.tx=mvcc;hsqldb.lob_file_scale
 
 To switch to a standalone database, see <<_perf_tuning_db>>.
 
+=== Tunning Linux for 15K nodes
+
+It is necessary to tune underlying Linux system, in case there will be plenty of nodes in Resource Manager.
+
+There should be big enough number of processes allowed, and number of open files as well.
+
+For 15.000 nodes, it is recommended to set `ulimit -u` bigger than 100.000 and `ulimit -n` at least 65.536.
+
+As well, we recommend to have at least 16GB RAM for 15.000 nodes.
```

Comment:
Replace "As well" with "In addition" or "Furthermore"

Reply:
Thank you